dashboard/app/views/levels/_level_group.haml

```diff
@@ -51,6 +51,8 @@
         = render partial: 'levels/single_text_match', locals: {standalone: false, level: level, last_attempt: last_attempt }
       - elsif level_class == "free_response"
         = render partial: 'levels/free_response', locals: {in_level_group: true, level: level, last_attempt: last_attempt}
+      - else
+        = raise "Unsupported level type #{level_class}"
```

Comment:
I only see one other instance of raising an exception in a page rendering in the rails app.. is this typical?  Does it just give the fail page when running on production?